Iss5 bio save btn

wibblywink opened the pull request:
This is just a Bootstrap button below the bio form. I left styling as-is because it will be impacted by others' design ideas.

Commit:
Added form field for bio. Added 'profile' header for page. Used template navbar as placeholder; will replace later.

Commit:
Added Save button for Bio form

tynanook commented:
@wibblywink @yuens1002 @JBigs206 
I like the button. 
I would add an action that returns the user to another page once the button is clicked.
Ideally, the user should be returned to the page they were on when when they clicked on their profile page.
This would mean the form should include the page to return to. For our case here, where we don't have dynamic behaviors in our html, I'm okay with the page the form returns to is the poems.html page or the index.html page.

You can checkout the review for in iss2 for an example. Basically, you need to assign an action. In iss2's case, the following is the syntax used, to load the 'poem.html' page after the button is selected.:
   <form action="./poem.html" method="get" role="form" class="form-horizontal">
    ...
    <button class="btn btn-default pull-right">Add my review</button>
  </form>

tynanook committed:
ty added changes, adding 'form action"

In the git hub interface, it didn't show me all the code around your form. I could only access the complete code form section around your button by selecting 'edit'. I added the <form action="./poems.html".... > line and the closing </form> line.

yuens1002 commented:
can you think about other info that might be applicable in the profile page?  see attached... 

![screen shot 2016-07-26 at 9 21 03 am](https://cloud.githubusercontent.com/assets/6596603/17146241/86ed705a-5312-11e6-8236-1db16fde8a60.png)

JBigs206 commented on profile.html:
```diff
@@ -0,0 +1,71 @@
+  </div>
+  <div class="save-btn col-xs-8 col-xs-offset-2 ">
+  <button type="button" class="btn btn-default">Save</button>
+  </div>
```
We could also look to implement a "cancel" button next to the save button that would clear out the text the user entered into the form if they decide at the last minute not to add it to their bio. This would be a implementation for a future sprint though not something that needs to be added to this issue as this is just about adding a save button.

tynanook commented on profile.html:
```diff
@@ -61,6 +61,10 @@ <h4 class="col-xs-8 col-xs-offset-2 col-md-6">About</h4>
       <textarea class="form-control" rows="3" placeholder="What's your story? What poems do you like?" required></textarea>
     </div>
   </div>
+  <div class="save-btn col-xs-8 col-xs-offset-2 ">
```
I recommend adding a comment describing what is being saved, or modifying the 'save-btn' class name to include what being saved, i.e., save-bio-btn.
